src/ubuntu/22.04/webassembly/Dockerfile

```diff
@@ -0,0 +1,72 @@
+FROM mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-22.04-coredeps-local
+
+# Dependencies for WebAssembly build
+RUN apt-get update \
+    && apt-get install -y \
+        libssl-dev \
+        libtinfo-dev \
+        locales \
+        nodejs \
+        node-gyp \
+        npm \
+        wget \
+        unzip \
+    && rm -rf /var/lib/apt/lists/*
+
+ENV CMAKE_VERSION=3.17
+RUN wget https://cmake.org/files/v${CMAKE_VERSION}/cmake-${CMAKE_VERSION}.0-Linux-x86_64.tar.gz \
+    && tar -xf cmake-${CMAKE_VERSION}.0-Linux-x86_64.tar.gz --strip 1 -C /usr/local \
+    && rm cmake-${CMAKE_VERSION}.0-Linux-x86_64.tar.gz
+
+# WebAssembly build needs working UTF-8 locale
+RUN locale-gen en_US.UTF-8
+
+# WebAssembly build needs typescript
+RUN npm i -g typescript
+
+# Install Emscripten toolchain
+ENV EMSCRIPTEN_VERSION=3.1.12
+ENV EMSCRIPTEN_PATH=/usr/local/emscripten
+ENV EMSDK_PATH=/usr/local/emscripten/emsdk
+
+RUN mkdir ${EMSCRIPTEN_PATH} \
+    && cd ${EMSCRIPTEN_PATH} \
+    && git clone https://github.com/emscripten-core/emsdk.git ${EMSDK_PATH} \
+    && cd ${EMSDK_PATH} \
+    && ./emsdk install ${EMSCRIPTEN_VERSION}-upstream \
+    && ./emsdk activate ${EMSCRIPTEN_VERSION}-upstream \
+    && chmod -R 777 ${EMSCRIPTEN_PATH}
+
+# Install V8 Engine
+SHELL ["/bin/bash", "-c"]
+
+ENV V8_VERSION=8.5.183
```

Comment:
Latest V8 is 11.1.218 but I don't know if we have specific preference. @radical @maraf

Reply:
Maybe the latest stable? like https://github.com/dotnet/dotnet-buildtools-prereqs-docker/pull/757

Reply:
Do I need to update also for 20.04/webassembly/Dockerfile?

Reply:
That would be good. are there yet another images?